installation/docker.md

```diff
@@ -4,3 +4,11 @@ The awesome folks at [LinuxServer.io](https://linuxserver.io) maintain a range o
 
 It is also possible to install LinuxServer Docker containers from the SSH console, which is also how to access the full [fleet of LinuxServer containers](https://fleet.linuxserver.io).
 
+## A note on Docker image architectures
```

Comment:
Container Architecture

Reply:
I'd say that the images really dictate where containers can be spun up from them, but ok NP :)